pubsub/topics/main.go

```diff
@@ -145,6 +154,43 @@ func publish(client *pubsub.Client, topic, msg string) error {
 	return nil
 }
 
+func publishThatScales(client *pubsub.Client, topic string, n uint64) error {
+	ctx := context.Background()
+	var wg sync.WaitGroup
+	var n_err uint64
+	// [START pubsub_publish_with_error_handling_that_scales]
+	t := client.Topic(topic)
```

Comment:
Nit: I would rename `t` to `topic` to help on readability, and the current `topic` (string) to `topicName`